blob/gcsblob/gcsblob.go

```diff
@@ -241,11 +241,11 @@ func (r *reader) Attributes() driver.ReaderAttributes {
 }
 
 func (r *reader) As(i interface{}) bool {
-	p, ok := i.(*storage.Reader)
+	p, ok := i.(**storage.Reader)
```

Comment:
We talked about this in person, looks fine.

You need to update the test though.

Reply:
Done.